pkg/assets/handlers.go

```diff
@@ -76,7 +77,7 @@ func CacheControlHandler(version string, h http.Handler) http.Handler {
 	})
 }
 
-func HTML5ModeHandler(h http.Handler) http.Handler {
+func HTML5ModeHandler(contextRoot string, h http.Handler) http.Handler {
```

Comment:
Add godoc describing what contextRoot must conform to (leading slash? trailing slash?)